### ISSUE TITLE
Crashes when DLSS is added to the plugin file

### ISSUE THREAD

**Issue by JumpStumpsMTB777** [praydog/REFramework#840]
REFramework works fine until I add the upscaler and dlss file to the plug in folder. It crashes as soon as reframework initializes.  

This is happening on every game except re4 remake. 

amd7745hx and 4060 mobile gpu (legion 5 pro)

[re2_framework_log.txt](https://github.com/praydog/REFramework/files/11858792/re2_framework_log.txt)

**Comment by PoloniumRain** [comment]
Same here. Don't know why this is even happening, as it was working a couple of months ago, then i try it today and it no longer works even though all the files are exactly the same. 

So i updated everything to the latest files and now it doesn't crash, but REFramework fails to initialize. Looks like loads of people are having crashing issues.

**Comment by geminihc** [comment]
same issue, and based on our logs (same as yours), its not loading DLSS eventhough we have latest drivers. not sure why or how to fix

**Comment by DaveACG** [comment]
I've been trying to diagnose an odd stutter issue with some of the RE games when run from a HDD, and I nabbed the cracked version to test against. The upscaler crashes when run with a cracked copy.

For the actual Steam version though, it loads fine. So if you're not running the actual release, that's perhaps the issue.

**Comment by stein3x** [comment]
I can confirm the DLSS plugin crashes the Upscaler/REFramework when running any RE games other than legit versions, on the contrary FSR2 & XeSS run fine

**Comment by PoloniumRain** [comment]
Yeah, it's only the cracked/DRM-free version that crashes. But why did DLSS work perfectly fine before, and then it started crashing? Before the crashing started i had played through almost all of Village with DLSS without any issues.

I'd expect this problem to affect a lot of people because Village had severe performance issues on release, due to Denuvo DRM. So even though i have a legit copy i had never touched it until i tested it a moment ago.

**Comment by DaveACG** [comment]
> Yeah, it's only the cracked/DRM-free version that crashes. But why did DLSS work perfectly fine before, and then it started crashing? Before the crashing started i had played through almost all of Village with DLSS without any issues.
> 
> I'd expect this problem to affect a lot of people because Village had severe performance issues on release, due to Denuvo DRM. So even though i have a legit copy i had never touched it until i tested it a moment ago.

To be somewhat pedantic, the performance issues weren't actually related to Denuvo, it was Capcom's own DRM. I get the hatred for Denuvo, but Capcom's DRM actually being the one at fault was spelled out in the initial cracks readme.

While that initial release was truly ridiculous in that it seemingly took all that media fervor for Capcom to finally take action, it was indeed fixed over 2 years ago (and that fix also got plenty of media attention). As for why it just started happening, who knows - changes in RE framework or the upscaler plugin could have triggered it, but you can't expect the developer to test against a cracked .exe. 

It's fine for this thread to let people know what the reason is, but it's not really an 'issue' when it's working fine with the legit copy.

**Comment by Oculdev** [comment]
I had this problem before. Latest drivers is not always the best. Just Use a different driver, I used the one from my laptop manufacturer, use dlss dll 2.5.1. Try 1080p in graphic settings, and lowest all other settings. Once it works, start to increase ur graphic settings from there. Dont forget to turn off fidelity fx in the graphic settings.

**Comment by kenneth84** [comment]
add SelfProtect=0 to steam_emu.ini

Should look like this:

Offline=0
SelfProtect=0

If you want to thank me, please take the time to tell others this fix when you encounter someone posting a similar issue. Cheers.

**Comment by thebennator** [comment]
you are a legend thank you so much

**Comment by AbzRaider** [comment]
> SelfProtect=0

Thanks a ton man this fixed it on the pirated version

**Comment by StarryNight54** [comment]
> SelfProtect=0

absolute legend
